### ISSUE TITLE
ips.usps.com: breakage

### ISSUE THREAD

**Issue by TopPercentageRattata** [uBlockOrigin/uAssets#25497]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://ips.usps.com/IPSWeb/verification_user_information.xhtml`

### Category

breakage

### Description

After logging in and updating some things, USPS site has a loading page where it says verifying identity that gets stuck at 99% when UBlock origin is running, at least on mobile, have not tested desktop. Some other people have mentioned this same issue on reddit in the past month, but no issue on the github about USPS site exists aside from another old issue on a different page whose attempted fix I happened to add to my filters but did not do anything for this either.
Reddit comments about this issue are here: https://old.reddit.com/r/usps_complaints/comments/1d6mwdu/error_signing_up_for_informed_delivery/lh60xpj/
Disabling UBlock Origin and refreshing page allows page to reach 100 and then move onto next page, so something is being blocked that shouldn't be.

### Other extensions used

Absolute Enable Right Click and Copy (for searching terms from pages that like to go 1984. Not the cause of USPS page not functioning.)
DeArrow(Only affects youtube thumbnails and titles)
TWP - Translate Web Pages (none set to default so does not affect this site)
OldLander (does not affect this site)
Old Reddit Redirect (does not affect this site)
Reddit Enhancement Suite (does not affect this site)

### Screenshot(s)

<details>
<summary>Screenshot of 99%</summary>

![Screenshot_20241002_121606_Firefox Nightly](https://github.com/user-attachments/assets/287456e7-2d04-418d-8ba0-6a244af5dda1)

What page shows when stuck at 99%.
</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Firefox Mobile: 133
filterset (summary):
 network: 151173
 cosmetic: 57015
 scriptlet: 21168
 html: 2002
listset (total-discarded, last-updated):
 default:
  user-filters: 1-0, never
  ublock-filters: 41065-132, now
  ublock-badware: 11252-6, now
  ublock-privacy: 1265-22, now
  ublock-unbreak: 2541-1, now
  ublock-quick-fixes: 134-13, now
  adguard-mobile: 9822-96, now
  easylist: 85669-260, now
  easyprivacy: 53116-71, now
  urlhaus-1: 24765-0, now
  plowe-0: 3546-1068, now
filterset (user): [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 494 ms (selfie)
 maxAssetCacheWait: 164 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 2
 network:
  google-analytics.com: 1
  googletagmanager.com: 1
```
</details>

**Comment by mapx-** [comment]
- open the uBO's logger: https://github.com/gorhill/uBlock/wiki/The-logger
- repeat the steps to get the issue
- go back into the uBO's logger and then copy and paste here all blocked requests

**Comment by TopPercentageRattata** [comment]
Sadly cannot do so because website does not trigger the verifying identity form for me as I already verified it the morning after reporting(Funnily enough trying to access the page when not required to has the USPS website generate an error code that is literally just the current time's timestamp)
Would need someone else to try set up USPS informed delivery for form to appear again, but most addresses in the US are eligible so hopefully shouldn't be too hard to find someone.

**Comment by mapx-** [comment]
@TopPercentageRattata  closing for now, when you'll be able to reproduce the issue again just follow the steps above and post the info here (without opening a new issue)

**Comment by TopPercentageRattata** [comment]
> @TopPercentageRattata closing for now, when you'll be able to reproduce the issue again just follow the steps above and post the info here (without opening a new issue)

Umm I won't be able to unless I literally change addresses IRL which might not happen for years. Almost anyone in USA can attempt to sign up for USPS informed delivery on the site and the issue will happen as seen with all the reddit comments on the issue

**Comment by TheMCNerd2017** [comment]
> @TopPercentageRattata closing for now, when you'll be able to reproduce the issue again just follow the steps above and post the info here (without opening a new issue)

Hello, I'm not the person that opened this issue but I am experiencing the same issue when trying to register for USPS informed delivery. I'm in medium mode and did a noop rule for the online-metrix domain being blocked, but the noop rule unfortunately does nothing for online-metrix since it is blocked by multiple filters. From what I was able to see in the logger, it is being blocked by the AdGuard Tracking Protection filter and Peter Lowe's Ad and tracking server list filter.

Here is a log of what is being blocked on the page in question (the Verification Device Profiling page):

<details><summary>Logger output</summary>

| | | | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/favicon.ico` |
| +1 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/rfRes/buttonBackgroundImage.png.xhtml?v=4.3.7.Final&db=eAFjZGBkZOBm!P-f8f!bV88Y!185f5yBCQBPWAk3&ln=org.richfaces.images` |
| +1 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/includes/rmin-step-3-lg.png` |
| +1 | \|\|online-metrix.net^ | -- | ips.usps.com | 3 | get | script | `https://h-usps.online-metrix.net/` | `aliasURL=https://content.usps.com/3bxidssz827zfjuc.js?jcvcydubtg6qkt2v=3fym673k&g84je89zl9h3hc79=07576D04EC7A-9da3fcb5-272d-4708-928e` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://content.usps.com/3bxidssz827zfjuc.js?jcvcydubtg6qkt2v=3fym673k&g84je89zl9h3hc79=07576D04EC7A-9da3fcb5-272d-4708-928e` |
| +1 | googletagmanager_gtm.js:5 | << | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +1 | \|\|googletagmanager.com/gtm.js$script,redirect-rule=googletagmanager_gtm.js:5 | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +1 | \|\|googletagmanager.com/gtm.js$script,3p,redirect-rule=googletagmanager_gtm.js,from=~bethesda.net\|~corsair.com\|~paddle.com\|~simon.com\|~olx.* | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +1 | * * 3p-script block | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +1 |  |  | ips.usps.com | 1 | get | xhr | `https://ips.usps.com/IPSWeb/resources/rest/getProperties` |
| +1 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/popupPanel.ecss.xhtml?db=eAHjW!XqPQAE!QKS&ln=org.richfaces` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelSizer.js.xhtml?ln=org.richfaces` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelBorders.js.xhtml?ln=org.richfaces` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanel.js.xhtml?ln=org.richfaces` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-event.js.xhtml` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/status.js.xhtml?ln=org.richfaces` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-base-component.js.xhtml` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-queue.js.xhtml` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces.js.xhtml` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jquery.js.xhtml` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jsf.js.xhtml?ln=javax.faces` |
| +1 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/skinning.ecss.xhtml?db=eAHjW!XqPQAE!QKS` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ips_gtm.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/analytics.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/verification_session_timeout.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/otpCommon.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/bootstrap.min.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery.mask.min.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery-1.12.4.min.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ipsCommon.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/device_reputation.js` |
| +1 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/usps_toolkit.js` |
| +0 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/verification_tmx_profiling.xhtml` |
</details>

**Comment by mapx-** [comment]
@TheMCNerd2017 repeat the test with uBO defaults, then post here the logger's content

**Comment by TheMCNerd2017** [comment]
> @TheMCNerd2017 repeat the test with uBO defaults, then post here the logger's content

Sorry for the late response (I was unable to check my email or github the last few days), I repeated the test with uBO defaults in a fresh separate Firefox profile and the same issue has occurred.

<details><summary>Logger output</summary>

| | | | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +4 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/favicon.ico` |
| +4 | \|\|online-metrix.net^ | -- | ips.usps.com | 3 | get | script | `https://h-usps.online-metrix.net/` | `aliasURL=https://content.usps.com/l554u2cxhjrj6hlh.js?suewpblgn3ogeg7s=3fym673k&dvzthg34vmlmcdhm=ECA01BB1DF5E-24e59676-5f22-499b-8c0d` |
| +4 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/5b4a262e-3342-44e2-8ad7-719998a68134.woff` |
| +4 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/4a9c62ab-b359-4081-8383-a0d1cdebd111.woff` |
| +4 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/d5af76d8-a90b-4527-b3a3-182207cc3250.woff` |
| +4 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/glyphicons-halflings-regular.woff2` |
| +4 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/hdr_bkg.png` |
| +4 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/rfRes/buttonBackgroundImage.png.xhtml?v=4.3.7.Final&db=eAFjZGBkZOBm!P-f8f!bV88Y!185f5yBCQBPWAk3&ln=org.richfaces.images` |
| +4 |  |  | ips.usps.com | 1 | get | script | `https://content.usps.com/l554u2cxhjrj6hlh.js?suewpblgn3ogeg7s=3fym673k&dvzthg34vmlmcdhm=ECA01BB1DF5E-24e59676-5f22-499b-8c0d` |
| +4 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/480.css` |
| +4 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/568.css` |
| +4 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/768.css` |
| +4 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/1024.css` |
| +3 | googletagmanager_gtm.js:5 | << | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +3 | \|\|googletagmanager.com/gtm.js$script,redirect-rule=googletagmanager_gtm.js:5 | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +3 | \|\|googletagmanager.com^ | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +3 |  |  | ips.usps.com | 1 | get | xhr | `https://ips.usps.com/IPSWeb/resources/rest/getProperties` |
| +3 |  |  | ips.usps.com | 3 | get | css | `https://use.fontawesome.com.cdn.cloudflare.net/` | `aliasURL=https://use.fontawesome.com/releases/v5.5.0/css/all.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://www.usps.com/global-elements/footer/css/footer-nofonts.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://www.usps.com/global-elements/footer/css/main-nofonts.css` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/ajax-loader-large-blue.gif` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/includes/rmin-step-3-lg.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/informed-delivery.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/logo-sb.jpg` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/popupPanel.ecss.xhtml?db=eAHjW!XqPQAE!QKS&ln=org.richfaces` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelSizer.js.xhtml?ln=org.richfaces` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelBorders.js.xhtml?ln=org.richfaces` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanel.js.xhtml?ln=org.richfaces` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-event.js.xhtml` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/status.js.xhtml?ln=org.richfaces` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-base-component.js.xhtml` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-queue.js.xhtml` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces.js.xhtml` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jquery.js.xhtml` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jsf.js.xhtml?ln=javax.faces` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/skinning.ecss.xhtml?db=eAHjW!XqPQAE!QKS` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/otp-navbar.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/usps-fonts.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/verification-otp.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/uspsCustom.css` |
| +3 |  |  | ips.usps.com | 3 | get | css | `https://use.fontawesome.com/releases/v5.5.0/css/all.css` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ips_gtm.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/analytics.js` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/bootstrap.min.css` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/verification_session_timeout.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/otpCommon.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/bootstrap.min.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery.mask.min.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery-1.12.4.min.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ipsCommon.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/device_reputation.js` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/usps_toolkit.js` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/ipsStyles_otp.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/cols.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/usps.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/responsivegridsystem.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/html5reset.css` |
| +3 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/verification_tmx_profiling.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/?LOA=1.5` |
| +2 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/registration.xhtml` |
| +1 |  |  | reg.usps.com | 1 | get | doc | `https://reg.usps.com/entreg/secure/InformedDeliveryValidateAction_input` |
| +1 |  |  | reg.usps.com | 1 | post | doc | `https://reg.usps.com/entreg/LoginIdAction` |
| +0 |  |  | reg.usps.com | 1 | get | image | `https://reg.usps.com/entreg/assets/images/ajax-loader-usps-white.gif` |
| +0 |  |  | reg.usps.com | 1 | post | xhr | `https://reg.usps.com/entreg/json/AuthenticateAction` |
</details>

**Comment by mapx-** [comment]
@TheMCNerd2017 

test

```adb
@@||googletagmanager.com/gtm.js$script,domain=ips.usps.com
```

**Comment by TheMCNerd2017** [comment]
> @TheMCNerd2017
> 
> test
> 
> ```
> @@||googletagmanager.com/gtm.js$script,domain=ips.usps.com
> ```

I went and added that to the 'My Filters' area and attempted the test again. The page still got stuck on the identity verification page at 99%. I even waited a few minutes and nothing else happened.

<details><summary>Logger output</summary>

| | | | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/favicon.ico` |
| +2 |  |  | ips.usps.com | 3 | get | script | `https://e15080.a.akamaiedge.net/` | `aliasURL=https://resources.digital-cloud-gov.medallia.com/wdcgov/2/onsite/generic1728577051306.js` |
| +2 |  |  | ips.usps.com | 3 | get | script | `https://resources.digital-cloud-gov.medallia.com/wdcgov/2/onsite/generic1728577051306.js` |
| +2 | \|\|googletagmanager.com^ | -- | ips.usps.com | 3 | get | frame | `https://www.googletagmanager.com/static/service_worker/4cc0/sw_iframe.html?origin=https%3A%2F%2Fips.usps.com` |
| +2 | googletagmanager_gtm.js:5 | << | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtag/js?id=G-3NXP3C8S9V&l=dataLayer&cx=c&gtm=45He4cc1v72384666za200` |
| +2 | \|\|googletagmanager.com/gtag/js$script,redirect-rule=googletagmanager_gtm.js:5 | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtag/js?id=G-3NXP3C8S9V&l=dataLayer&cx=c&gtm=45He4cc1v72384666za200` |
| +2 | \|\|googletagmanager.com^ | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtag/js?id=G-3NXP3C8S9V&l=dataLayer&cx=c&gtm=45He4cc1v72384666za200` |
| +2 |  |  | ips.usps.com | 3 | get | script | `https://e15080.a.akamaiedge.net/` | `aliasURL=https://resources.digital-cloud-gov.medallia.com/wdcgov/2/onsite/embed.js` |
| +2 |  |  | ips.usps.com | 3 | get | script | `https://resources.digital-cloud-gov.medallia.com/wdcgov/2/onsite/embed.js` |
| +2 | googletagmanager_gtm.js:5 | << | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtag/js?id=G-74117DKSEN&l=dataLayer&cx=c&gtm=45He4cc1v72384666za200` |
| +2 | \|\|googletagmanager.com/gtag/js$script,redirect-rule=googletagmanager_gtm.js:5 | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtag/js?id=G-74117DKSEN&l=dataLayer&cx=c&gtm=45He4cc1v72384666za200` |
| +2 | \|\|googletagmanager.com^ | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtag/js?id=G-74117DKSEN&l=dataLayer&cx=c&gtm=45He4cc1v72384666za200` |
| +2 | *$ping,3p | -- | ips.usps.com | 3 | post | beacon | `https://www.google.com/ccm/collect?en=page_view&dr=reg.usps.com&dl=https%3A%2F%2Fips.usps.com%2FIPSWeb%2Fverification_tmx_profiling.xhtml&scrsrc=www.googletagmanager.com&frm=0&rnd=908701923.1734419841&dt=USPS.com%C2%AE%C2%A0-%C2%A0Verification%20Device%20Profiling&auid=382408875.1734419527&navt=r&npa=0&gtm=45He4cc1v72384666za200&gcd=13l3l3l3l1l1&dma=0&tag_exp=101925629~102067555~102067808~102081485~102198178&tft=1734419840616&tfd=2470&apve=1` |
| +2 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/5b4a262e-3342-44e2-8ad7-719998a68134.woff` |
| +2 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/4a9c62ab-b359-4081-8383-a0d1cdebd111.woff` |
| +2 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/d5af76d8-a90b-4527-b3a3-182207cc3250.woff` |
| +2 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/glyphicons-halflings-regular.woff2` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/hdr_bkg.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/footer-logo.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/rfRes/buttonBackgroundImage.png.xhtml?v=4.3.7.Final&db=eAFjZGBkZOBm!P-f8f!bV88Y!185f5yBCQBPWAk3&ln=org.richfaces.images` |
| +2 | \|\|online-metrix.net^ | -- | ips.usps.com | 3 | get | script | `https://h-usps.online-metrix.net/` | `aliasURL=https://content.usps.com/7gp218lqj7b0cnkn.js?uhydvhkmcewycyfv=3fym673k&f63j5fib5o814utw=646E75625E6C-67d25a21-e977-4720-bb85` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://content.usps.com/7gp218lqj7b0cnkn.js?uhydvhkmcewycyfv=3fym673k&f63j5fib5o814utw=646E75625E6C-67d25a21-e977-4720-bb85` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/480.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/568.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/768.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/1024.css` |
| +2 | @@\|\|googletagmanager.com/gtm.js$script,from=ips.usps.com | ++ | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +2 |  |  | ips.usps.com | 1 | get | xhr | `https://ips.usps.com/IPSWeb/resources/rest/getProperties` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/youtube_logo.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/x_logo.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/threads_logo.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/pinterest_logo.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/instagram_logo.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/facebook_logo.png` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://www.usps.com/global-elements/footer/css/footer-nofonts.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://www.usps.com/global-elements/footer/css/main-nofonts.css` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/ajax-loader-large-blue.gif` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/includes/rmin-step-3-lg.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/informed-delivery.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/logo-sb.jpg` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/popupPanel.ecss.xhtml?db=eAHjW!XqPQAE!QKS&ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelSizer.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelBorders.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanel.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-event.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/status.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-base-component.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-queue.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jquery.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jsf.js.xhtml?ln=javax.faces` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/skinning.ecss.xhtml?db=eAHjW!XqPQAE!QKS` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/otp-navbar.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/usps-fonts.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/verification-otp.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/uspsCustom.css` |
| +2 |  |  | ips.usps.com | 3 | get | css | `https://use.fontawesome.com.cdn.cloudflare.net/` | `aliasURL=https://use.fontawesome.com/releases/v5.5.0/css/all.css` |
| +2 |  |  | ips.usps.com | 3 | get | css | `https://use.fontawesome.com/releases/v5.5.0/css/all.css` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ips_gtm.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/analytics.js` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/bootstrap.min.css` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/verification_session_timeout.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/otpCommon.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/bootstrap.min.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery.mask.min.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery-1.12.4.min.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ipsCommon.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/device_reputation.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/usps_toolkit.js` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/ipsStyles_otp.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/cols.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/usps.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/responsivegridsystem.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/html5reset.css` |
| +0 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/verification_tmx_profiling.xhtml` |
</details>

**Comment by mapx-** [comment]
keep the filter above and test also:

`@@||googletagmanager.com/gtag/js$script,domain=ips.usps.com`

if still broken add also and test:
`@@||h-usps.online-metrix.net$script,domain=ips.usps.com`

**Comment by mapx-** [comment]
ok, now keep only the last filter and test again

**Comment by mapx-** [comment]
@ryanbr 

```adb
@@||h-usps.online-metrix.net$script,domain=ips.usps.com
```

**Comment by ryanbr** [comment]
`usps.com##+js(acis, tmx_post_session_params_fixed)` ?

Allowing online-metrix.net will open users to port scans. https://www.bleepingcomputer.com/news/security/ebay-port-scans-visitors-computers-for-remote-access-programs/

**Comment by mapx-** [comment]
@TheMCNerd2017 could you test only ryan's filter above ?

**Comment by TheMCNerd2017** [comment]
> @TheMCNerd2017 could you test only ryan's filter above ?

I tested ryan's filter, and the page got stuck at 99% and would not continue.

<details><summary>Logger output</summary>

| | | | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/favicon.ico` |
| +3 | \|\|online-metrix.net^ | -- | ips.usps.com | 3 | get | script | `https://h-usps.online-metrix.net/` | `aliasURL=https://content.usps.com/k43m08tbkb5qny95.js?vlt46s9486yg8b3h=3fym673k&lpskhy5f6iaoerlw=DF5673111F1D-edf31153-d0a8-4c45-a45f` |
| +3 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/5b4a262e-3342-44e2-8ad7-719998a68134.woff` |
| +3 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/4a9c62ab-b359-4081-8383-a0d1cdebd111.woff` |
| +3 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/usps/d5af76d8-a90b-4527-b3a3-182207cc3250.woff` |
| +3 |  |  | ips.usps.com | 1 | get | font | `https://ips.usps.com/fonts/glyphicons-halflings-regular.woff2` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/hdr_bkg.png` |
| +3 |  |  | ips.usps.com | 3 | get | image | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/images/footer-logo.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/footer-logo.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/rfRes/buttonBackgroundImage.png.xhtml?v=4.3.7.Final&db=eAFjZGBkZOBm!P-f8f!bV88Y!185f5yBCQBPWAk3&ln=org.richfaces.images` |
| +3 |  |  | ips.usps.com | 1 | get | script | `https://content.usps.com/k43m08tbkb5qny95.js?vlt46s9486yg8b3h=3fym673k&lpskhy5f6iaoerlw=DF5673111F1D-edf31153-d0a8-4c45-a45f` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/480.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/568.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/768.css` |
| +3 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/1024.css` |
| +3 | googletagmanager_gtm.js:5 | << | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +3 | \|\|googletagmanager.com/gtm.js$script,redirect-rule=googletagmanager_gtm.js:5 | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +3 | \|\|googletagmanager.com^ | -- | ips.usps.com | 3 | get | script | `https://www.googletagmanager.com/gtm.js?id=GTM-MVCC8H` |
| +3 |  |  | ips.usps.com | 1 | get | xhr | `https://ips.usps.com/IPSWeb/resources/rest/getProperties` |
| +3 |  |  | ips.usps.com | 3 | get | image | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/images/youtube_logo.png` |
| +3 |  |  | ips.usps.com | 3 | get | image | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/images/x_logo.png` |
| +3 |  |  | ips.usps.com | 3 | get | image | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/images/threads_logo.png` |
| +3 |  |  | ips.usps.com | 3 | get | image | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/images/pinterest_logo.png` |
| +3 |  |  | ips.usps.com | 3 | get | image | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/images/instagram_logo.png` |
| +3 |  |  | ips.usps.com | 3 | get | image | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/images/facebook_logo.png` |
| +3 |  |  | ips.usps.com | 3 | get | css | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/css/footer-nofonts.css` |
| +3 |  |  | ips.usps.com | 3 | get | css | `https://cs1799.wpc.upsiloncdn.net/` | `aliasURL=https://www.usps.com/global-elements/footer/css/main-nofonts.css` |
| +3 |  |  | ips.usps.com | 3 | get | css | `https://use.fontawesome.com.cdn.cloudflare.net/` | `aliasURL=https://use.fontawesome.com/releases/v5.5.0/css/all.css` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/youtube_logo.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/x_logo.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/threads_logo.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/pinterest_logo.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/instagram_logo.png` |
| +3 |  |  | ips.usps.com | 1 | get | image | `https://www.usps.com/global-elements/footer/images/facebook_logo.png` |
| +2 | ##+js(abort-current-script, tmx_post_session_params_fixed) |  | ips.usps.com |  | get | scriptlet | `https://ips.usps.com/IPSWeb/verification_tmx_profiling.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://www.usps.com/global-elements/footer/css/footer-nofonts.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://www.usps.com/global-elements/footer/css/main-nofonts.css` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/ajax-loader-large-blue.gif` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/IPSWeb/includes/rmin-step-3-lg.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/informed-delivery.png` |
| +2 |  |  | ips.usps.com | 1 | get | image | `https://ips.usps.com/images/logo-sb.jpg` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/popupPanel.ecss.xhtml?db=eAHjW!XqPQAE!QKS&ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelSizer.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanelBorders.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/popupPanel.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-event.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/status.js.xhtml?ln=org.richfaces` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-base-component.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces-queue.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/richfaces.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jquery.js.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/IPSWeb/javax.faces.resource/jsf.js.xhtml?ln=javax.faces` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/IPSWeb/rfRes/skinning.ecss.xhtml?db=eAHjW!XqPQAE!QKS` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/otp-navbar.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/usps-fonts.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/verification-otp.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/uspsCustom.css` |
| +2 |  |  | ips.usps.com | 3 | get | css | `https://use.fontawesome.com/releases/v5.5.0/css/all.css` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ips_gtm.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/analytics.js` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/bootstrap.min.css` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/verification_session_timeout.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/otpCommon.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/bootstrap.min.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery.mask.min.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/jquery-1.12.4.min.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/ipsCommon.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/device_reputation.js` |
| +2 |  |  | ips.usps.com | 1 | get | script | `https://ips.usps.com/js/usps_toolkit.js` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/ipsStyles_otp.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/cols.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/usps.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/responsivegridsystem.css` |
| +2 |  |  | ips.usps.com | 1 | get | css | `https://ips.usps.com/css/global/html5reset.css` |
| +2 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/verification_tmx_profiling.xhtml` |
| +2 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/?LOA=1.5` |
| +2 |  |  | ips.usps.com | 1 | get | doc | `https://ips.usps.com/IPSWeb/registration.xhtml` |
| +1 |  |  | reg.usps.com | 1 | get | doc | `https://reg.usps.com/entreg/secure/InformedDeliveryValidateAction_input` |
| +1 |  |  | reg.usps.com | 1 | post | doc | `https://reg.usps.com/entreg/LoginIdAction` |
| +0 |  |  | reg.usps.com | 1 | get | image | `https://reg.usps.com/entreg/assets/images/ajax-loader-usps-white.gif` |
| +0 |  |  | reg.usps.com | 1 | post | xhr | `https://reg.usps.com/entreg/json/AuthenticateAction` |
</details>
